packages/react-components/react-calendar-compat/library/src/utils/dateMath/dateMath.ts

```diff
@@ -165,6 +165,7 @@ export function getDateRangeArray(
   firstDayOfWeek: DayOfWeek,
   workWeekDays?: DayOfWeek[],
   daysToSelectInDayView: number = 1,
+  reverse?: boolean,
```

Comment:
I don't like the idea of adding another parameter to this function as it's already pretty complex. Maybe it would be good to split it to get rid of so many parameters. But currently I did it according to existing style